lib/utilities/is-live-reload-request.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function isLiveReloadRequest(url) {
+  let regex = /\/livereload$/gi;
```

Comment:
This should receive the live reload path command line option, and use that to match absolute path. We should not need a regexp match here.

Reply:
There is no current command line option to take `livereload request path`, Are we going to provide a new one?

Reply:
Oh. Purpose of this option is different than what you pointed above. This path where `livereload server` is mounted with respect to `ember's express server`. This is the location from where `livereload.js` is served not the `upgrade` request. `livereload.js` running in the client(browser) is completely unaware of this option. If we are going to use this option as `path` to `livereload` requests, we have to set `host` and `port` in `ember-cli`  or in [ember-cli-inject-livereload](https://github.com/ember-cli/ember-cli-inject-live-reload/blob/master/index.js#L19) as well, because of the [this check](https://github.com/livereload/livereload-js/blob/master/dist/livereload.js#L314) in `livereload.js`.

Reply:
> Purpose of this option is different than what you pointed above. This path where livereload server is mounted with respect to ember's express server. 

Yep, you are absolutely describing the current situation, I am saying that is wrong.

> livereload.js running in the client(browser) is completely unaware of this option.

Yep, I'm with you here. 

> If we are going to use this option as path to livereload requests, we have to set host and port in ember-cli or in ember-cli-inject-livereload as well, because of the this check in livereload.js.

We already do set the host and port in `ember-cli-inject-live-reload` [here](https://github.com/ember-cli/ember-cli-inject-live-reload/blob/68c237eca6cefc760a97ab30f37c215c84bd7fc7/index.js#L35). I agree that we need to properly set `path` in that query string as well (though it will need to use the current URL + the `liveReloadPrefix`).

---

The point of `liveReloadPrefix` is to scope **_all_** of the live reload related things including but not limited to:

* loading the `ember-cli-inject-live-reload.js` asset from the ember-cli express server
* loading the actual `livereload.js` asset from the tiny-lr server
* handling the upgrade requests from the live reload client

Reply:
Got it. Thank you for the detailed explanation. I will make the changes and update the PR as soon as possible.